build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+# Build our server
+
+npm run build
```

Comment:
Only build if you need to by wrapping this in a conditional.

Reply:
Build must be ran before any testing because there may be changes in the source code that haven't been applied in the current dist directory. 

`npm run build` replaces the contents of the dist directory with the updated files. If you wrap this in a conditional, looking for the presence of a dist directory, you will not be able to test against the current source of truth, only on "some source of truth that once existed"